new-pipeline/src/evo.js

```diff
@@ -1021,7 +1028,7 @@ function saveStateToUrlAndCookie() {
 
   // If advanced settings are not at their defaults, display a notice in the panel header
   if (gInitialPageState.use_submission_date !== 0 || gInitialPageState.sanitize !==
```

Comment:
Default sanitize setting is to sanitize, so this should probably be left more or less the same.